sandbox/lua/encoders/schema_influx_write.lua

```diff
@@ -129,12 +130,17 @@ require "string"
 local interp = require "msg_interpolate"
 
 local name_prefix_orig  = read_config("name_prefix") or "name"
-local name_prefix = name_prefix_orig
```

Comment:
By getting rid of this variable definition you've made the scope of the `name_prefix` defined below to be global. You'll want to at least keep a `local name_prefix` declaration here.